.gitlab-ci.yml

```diff
@@ -138,6 +138,7 @@ convert_raspbian_raspberrypi4:
 
 .template_test_acceptance: &test_acceptance
   stage: test_acceptance
+  when: manual
```

Comment:
Why?

Reply:
For testing :)

Reply:
Saves me from stopping each one, each time.